lib/Frontend/ModuleInterfaceSupport.cpp

```diff
@@ -69,15 +69,24 @@ static void printToolVersionAndFlagsComment(raw_ostream &out,
     out << " -module-alias " << MODULE_DISAMBIGUATING_PREFIX <<
            moduleName << "=" << moduleName;
 
-    SmallVector<ImportedModule> imports;
-    M->getImportedModules(imports,
-                          {ModuleDecl::ImportFilterKind::Default,
+    ModuleDecl::ImportFilter filter = {ModuleDecl::ImportFilterKind::Default,
                            ModuleDecl::ImportFilterKind::Exported,
-                           ModuleDecl::ImportFilterKind::SPIOnly,
-                           ModuleDecl::ImportFilterKind::SPIAccessControl});
+                           ModuleDecl::ImportFilterKind::SPIAccessControl};
+    if (Opts.PrintPrivateInterfaceContent)
+      filter |= ModuleDecl::ImportFilterKind::SPIOnly;
+
+    SmallVector<ImportedModule> imports;
+    M->getImportedModules(imports, filter);
     M->getMissingImportedModules(imports);
+
     for (ImportedModule import: imports) {
       StringRef importedName = import.importedModule->getNameStr();
+      // Skip Swift as it's commonly used in inlinable code,
+      // and Builtin as it's imported implicitly by name.
```

Comment:
I'm also guessing `Builtin` is written in a lot of `@inlinable`/`@_transparent` function bodies.